sdc-plugin/tests/counter/counter.tcl

```diff
@@ -23,4 +23,6 @@ close $fh
 
 # Write out the SDC file after the clock propagation step
 write_sdc [test_output_path "counter.sdc"]
+
+yosys proc
```

Comment:
So all process's should've been converted at line 10.  The fact that we have some again means something else is wrong.

Reply:
Unfortunately, the Yosys error message would not tell anything about the internal name or origin of the process it can't export.

But if this is something that should've already been converted above, I have to leave it to the plugin-owner to figure that out :)

Reply:
I'll pass this then on to @tmichalak to investigate why processes have not been converted. Should this be a bug ?

Reply:
Given that the original error is in a library cell-ish thing (e.g. `$paramod\FDRE\INIT=1'0`), I strongly suspect this is just a case where yosys itself generated the process post-proc.  But I'm not sure.

Reply:
it is an init process, so if `synth_xilinx` should've baked that into the netlist and removed the process then it should not be there anymore.

I could also imagine that some platforms might not need do do anything when the initialization is to all bits zero, and thus leaves the process in there ? I'll have to investigate. There were some of these things addressed in Yosys, maybe this was left out ?

I'll investigate (but low prio, probably will only have time later this evening)

Reply:
So while it might be an init process, if that is a library cell, then the process is "internal", similar to the processes hidden with https://github.com/YosysHQ/yosys/pull/2679

Reply:
Yes, that is the change I remembered when I said 'some of these things addressed in Yosys'